umap/static/umap/js/umap.layer.js

```diff
@@ -677,7 +677,7 @@ U.DataLayer = L.Evented.extend({
     this._loading = true
     const [geojson, response, error] = await this.map.server.get(this._dataUrl())
     if (!error) {
-      this._last_modified = response.headers.get('last-modified')
+      this._reference_version = response.headers.get('X-Umap-Version')
```

Comment:
nit: `umap-version` seems a bit like it's the version of uMap itself. What about `x-Data-Version` or `X-DataLayer-Version` ?

Reply:
I went with `X-DataLayer-Version` and `X-DataLayer-Reference`.